src/reanimated2/animation/sequence.ts

```diff
@@ -58,6 +59,14 @@ export function withSequence(
     });
   }
 
+  const initialAnimationBounds: AnimationBounds[] = _animations.map((anim) => {
+    const a = typeof anim === 'function' ? anim() : anim;
+    return {
+      start: a.startValue,
+      end: a.toValue,
+    };
+  });
```

Comment:
Please don't use one-letter-names:
```suggestion
  const initialAnimationBounds: AnimationBounds[] = _animations.map((animationObject) => {
    const animation = typeof animationObject === 'function' ? animationObject() : animationObject;
    return {
      start: animation.startValue,
      end: animation.toValue,
    };
  });
```